src/metrics/utils/utils.go

```diff
@@ -78,3 +81,36 @@ func GetTimeMiliseconds() int64 {
 
 	return millis
 }
+
+func GetIPfromMultiaddress(multiaddr string) (ip string, err error) {
```

Comment:
do we still need this?

Reply:
As we saw, there are some IPs shared on the ENR that belong to a restringed or a private range, I would still use this function to extract the real IP from the established connection

Reply:
out of curiosity, have you seen any enr containing an ip in the range `192.168.x.x` for example? I remember we saw this 100.101... but have we seen any of the typical private network ones? i doubt it but just curious.

Reply:
Noup, I didn't see so far any ENR returning private networks such as 127.0... or 192.168... , I just saw restringed IPs